FlowCal/io.py

```diff
@@ -1527,8 +1527,16 @@ def __new__(cls, infile):
         detector_voltage = tuple(detector_voltage)
 
         # Amplifier gain: Stored in the keyword parameter $PnG for channel n.
-        amplifier_gain = [fcs_file.text.get('$P{}G'.format(i))
-                          for i in range(1, num_channels + 1)]
+        # The FlowJo Collector's Edition software saves the amplifier gain in
+        # keyword parameters CytekP01G, CytekP02G, CytekP03G, ... for channels
+        # 1, 2, 3, ...
+        if 'CREATOR' in fcs_file.text and \
+                'FlowJoCollectorsEdition' in fcs_file.text.get('CREATOR'):
+            amplifier_gain = [fcs_file.text.get('CytekP{:02d}G'.format(i))
+                              for i in range(1, num_channels + 1)]
+        else:
+            amplifier_gain = [fcs_file.text.get('$P{}G'.format(i))
+                              for i in range(1, num_channels + 1)]
```

Comment:
Could maybe check if `$PnG` keywords exist first? (are not `None`) and default to those first? Hard to say what the intention is if `$PnG` and `CytekPnnG` exist and `CREATOR` contains `FlowJoCollectorsEdition`, though. We also didn't do this for `detector_voltage` above (if `CREATOR` contains `CellQuest Pro`, we went straight to the `BD$WORDn` keywords).